### ISSUE TITLE
Add an interface version to control the push and overwrite of the interface.

### ISSUE THREAD

**Issue by timnick-snow** [TongchengOpenSource/smart-doc#427]
## Summary(关于这个pr的描述)
It is recommended to add extra tags to describe the interface version, which could control the overwrite issue of the interfaces.
The actual problem encountered is: When multiple team members are developing on different branches, the pushed interfaces can easily overwrite each other, and the final documentation cannot collect a substantial summary of the latest interfaces.

建议增加额外的tag，来描述接口的版本，这样就能控制接口的覆盖问题。
遇到的实际问题是：团队有多个人在不同的分支进行开发，推送的接口容易造成相互覆盖，最终呈现的文档不能收集实质上的最新接口的汇总。
```java
    /**
     * 
     *
     * @param param   
     * @apiNote obtain something
     * @version 1.2
     */
    @GetMapping("/some/path/to/get")
    public CommonResult<SomeObject> getSomething(String param) {
        
    }
```

**Comment by timnick-snow** [comment]
Of course, a more significant issue is that the biggest difference between different branches is actually the number of interfaces.
An interface added in branch A doesn't exist in branch B, so when pushing from branch B a second time, the newly added interface is gone. There is no conflict of interfaces at this point; it's just a simple merge, and this feature should be relatively straightforward, right?
当然一个更加重要的问题是，由于不同的分支最大的不同其实还是，接口的数量就不一样。
在A分支新增的接口由于在B分支没有，当第二次在B分支推送时，新增的接口也没了。此时并不存在接口冲突。只是简单的合并，这个功能应该会比较简单吧

**Comment by shalousun** [comment]
@timnick-snow Pull requests are welcome for collaborative development.
